test-libbz2-rs-sys/Cargo.toml

```diff
@@ -14,3 +14,7 @@ edition.workspace = true
 bzip2-sys = { version = "0.1.11", features = ["static"] }
 libbz2-rs-sys = { workspace = true, default-features = true, features = ["testing-prefix"] }
 libc.workspace = true
+
+[dev-dependencies]
+tempfile = "3.13.0"
+crc32fast = "=1.4.2"
```

Comment:
These don't seem to be used anywhere.

Reply:
they are actually, by `tests/quick.rs` and friends

Reply:
ah but not here, right, I just discovered that too.